packages/app/src/providers/h5grove/utils.test.ts

```diff
@@ -1,100 +1,69 @@
-import { DTypeClass, Endianness } from '@h5web/shared/hdf5-models';
+import { Endianness } from '@h5web/shared/hdf5-models';
+import {
+  boolType,
+  compoundType,
+  cplxType,
+  floatType,
+  intType,
+  strType,
+  unknownType,
+} from '@h5web/shared/hdf5-utils';
 
 import { convertH5GroveDtype } from './utils';
 
 describe('convertH5GroveDtype', () => {
   it('should convert integer dtypes', () => {
-    expect(convertH5GroveDtype('<i4')).toEqual({
-      class: DTypeClass.Integer,
-      size: 32,
-      endianness: Endianness.LE,
-    });
-    expect(convertH5GroveDtype('>u8')).toEqual({
-      class: DTypeClass.Unsigned,
-      size: 64,
-      endianness: Endianness.BE,
-    });
+    expect(convertH5GroveDtype('<i4')).toStrictEqual(
+      intType(32, false, Endianness.LE),
```

Comment:
Not a huge fan of the fact we have to inspect `intType` to know what this `false/true` is for.

Add a helper `uintType` instead and keep `intType` for signed integers ?

Reply:
I hesitated but you're right, let's do that.